Database/Table Scripts/CreatePetTableScript.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE dbo.Pet(
```

Comment:
Check out what I did in the other Create table scripts. I checked if table exists and if so, drop and then re-create. do that here too.